lib/gollum/app.rb

```diff
@@ -305,8 +305,8 @@ def wiki_new
       if settings.wiki_options[:template_page] then
         temppage = wiki_page("/_Template")
         @template_page = (temppage.page != nil) ? temppage.page.raw_data : "Template page option is set, but no /_Template page is present or committed."
-      end          
-      wikip = wiki_page(params[:splat].first.gsub('+', '-'))
```

Comment:
This `gsub` was added because apparently the `params` would contain strings like 'New+Page', with `+` characters instead of spaces. Tests show that `params` are now already properly sanitized by sinatra, so we get `New Page` and don't want to change that.